boards/arm/rp2040/common/src/rp2040_pwmdev.c

```diff
@@ -0,0 +1,103 @@
+/****************************************************************************
+ * boards/arm/rp2040/common/src/rp2040_pwm.c
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+#include <stdio.h>
+#include <debug.h>
+#include <errno.h>
+
+#include "rp2040_pwm.h"
+
+// #include <stdint.h>
+// #include <stdbool.h>
+// #include <debug.h>
+// #include <assert.h>
+
+// #include <nuttx/board.h>
+// #include <arch/board/board.h>
+// #include <nuttx/arch.h>
+// #include <nuttx/irq.h>
+// #include <arch/irq.h>
+// #include <nuttx/signal.h>
+
```

Comment:
Shouldn't use C++ comments here; recommend to remove unneeded `#include` directives instead.

Reply:
Thanks, I meant to delete these but missed them.